test/tests/meshgenerators/plane_deletion/plane_deletion.i

```diff
@@ -1,5 +1,20 @@
 [Mesh]
-  type = MeshGeneratorMesh
+  [MeshGenerators]
```

Comment:
This is an extra layer of the input compared with our final goal.

Reply:
What? What do you think our final goal is?

Reply:
Something like this:
```
[Mesh]
   [generator]
   []
   [modifier1]
   []
   [modifier2]
   []
[]
```

Reply:
Maybe... I'll talk it over with the @idaholab/moose-team. We have Partitioners under the Mesh section and we've had other things under here before. Actually Mesh/Generation was a previous section many years ago! Funny how we've come full circle

Reply:
I believe we are going to go with the nesting... Except I did drop "Mesh" from the inner level.